app/dao/notifications_dao.py

```diff
@@ -408,3 +413,34 @@ def get_april_fools(year):
     """
     return pytz.timezone('Europe/London').localize(datetime(year, 4, 1, 0, 0, 0)).astimezone(pytz.UTC).replace(
         tzinfo=None)
+
+
+def get_total_sent_notifications_in_date_range(start_date, end_date, notification_type):
+    result = db.session.query(
+        func.count(NotificationHistory.id).label('count')
+    ).filter(
+        NotificationHistory.key_type != KEY_TYPE_TEST,
+        NotificationHistory.status.in_(NOTIFICATION_STATUS_TYPES_COMPLETED),
+        NotificationHistory.created_at >= start_date,
+        NotificationHistory.created_at <= end_date,
+        NotificationHistory.notification_type == notification_type
+    ).first()
+
+    return 0 if result is None else result.count
+
+
+def get_total_sent_notifications_yesterday():
+    today = datetime.utcnow()
+    start_date = get_midnight_for_day_before(today)
+    end_date = get_midnight_for_date(today)
+
+    return {
+        "start_date": start_date,
+        "end_date": end_date,
+        "email": {
+            "count": get_total_sent_notifications_in_date_range(start_date, end_date, 'email')
+        },
+        "sms": {
+            "count": get_total_sent_notifications_in_date_range(start_date, end_date, 'sms')
```

Comment:
actually it might not be they're not hugely long running queries (yesterday's data set of ~39k messages on live takes 42 + 24ms, whereas with a groupby it takes 45ms )

since it only saves 20ms on a scheduled task that runs at unsavoury hours i'm not bothered, but its just something to keep in mind - grouping is good